src/middlewared/middlewared/plugins/smb.py

```diff
@@ -1476,7 +1476,7 @@ def get_acl_type(sb_info):
             return
 
         current_acltype = get_acl_type(this_mnt['super_opts'])
-        child_mounts = filter_list(list(mntinfo.values()), [['mountpoint', '^', path]])
+        child_mounts = filter_list(list(mntinfo.values()), [['mountpoint', '^', f'{path}/']])
```

Comment:
This is probably fine, although if path already has a trailing "/" would this complicate things?

Could always do
`child_mounts = filter_list(list(mntinfo.values()), [['mountpoint', '^', path if path.endswith('/') else f'{path}/']])
`

Reply:
Good point. I switched to using `Path` from our schema validators, which normalizes the path with `os.path.normpath()`.